wca-regulations.md

```diff
@@ -492,6 +492,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - E3c) Stickers (self-supplied).
     - E3d) Stopwatch or watch (self-supplied) for keeping track of the elapsed time, if it is approved by the WCA Delegate.
     - E3e) Other non-electronic aids that do not give an unfair advantage, at the discretion of the WCA Delegate (see [Regulation 2i1](regulations:regulation:2i1)).
+    - E3f) Tools to remove markings (self-supplied) such as correction fluid, correction tape, or erasers.
```

Comment:
@lgarron @Jambrose777 Do you think this wording works fine? I think it solves Lucas' issue (although this wording doesn't exist in the regulations).

IMO, it's important enough to make clear what these tools are in the regulations rather than the guidelines for clarity, so I think using a completely new wording rather than the guidelines is important enough here.

Reply:
I think this works well!

I'd add a tiny comma, although it's not strictly needed:
```suggestion
    - E3f) Tools to remove markings (self-supplied), such as correction fluid, correction tape, or erasers.
```

Reply:
I think it looks better with the comma - I'll commit the suggestion